stories/Conditional.stories.js

```diff
@@ -1,31 +1,28 @@
-import React, { Fragment } from 'react'
+import React from 'react'
 
 import { storiesOf } from '@storybook/react'
 
-import { ManagedForm, File } from '../src'
-import { Conditional } from '../src/helper/Conditional.js'
+import { File, Conditional, clear } from '../src/index.js'
+import { createFormDecorator } from './helpers/formDecorator.js'
 
-storiesOf('Conditional fields', module).add('Default', () => (
-    <ManagedForm onSubmit={v => console.log(JSON.stringify(v, null, 2))}>
-        {({ values }) => (
-            <Fragment>
-                <File name="file1" label="File one" />
+storiesOf('Conditional fields', module)
+    .addDecorator(createFormDecorator({ mutators: { clear } }))
+    .add('Default', ({ formProps }) => (
+        <>
+            <File name="file1" label="File one" />
```

Comment:
Suggestions:
We could use a `RadioGroup` here.
Then you don't have to select a file to see the behavior